pkg/gardenlet/controller/managedseed/valueshelper_test.go

```diff
@@ -224,8 +222,8 @@ var _ = Describe("ValuesHelper", func() {
 					},
 				},
 				FeatureGates: map[string]bool{
-					string(features.ReversedVPN): false,
-					string(features.HVPA):        true,
+					string("FooFeature"): true,
+					string("BarFeature"): true,
```

Comment:
To preserve the exact test case that is described in https://github.com/gardener/gardener/pull/7167#discussion_r1053013003

```suggestion
					"FooFeature": false,
					"BarFeature": true,
```